pennylane_lightning/lightning_qubit.py

```diff
@@ -160,6 +160,12 @@ class LightningQubit(QubitDevice):
             the expectation values. Defaults to ``None`` if not specified. Setting
             to ``None`` results in computing statistics like expectation values and
             variances analytically.
+        mcmc (bool): Determine whether we use Markov Chain Monte Carlo sampling method when generating samples.
+        kernel_name (str): name of transition kernel. Current version supports two kernels: ``Local`` and ``NonZeroRandom``.
+            The ``Local`` kernel conducts a ``SpinFlip`` local transition between states. The ``Local`` kernel generates a
+            random qubit site and then generates a random number to determine the new bit at that qubit site. The ``NonZeroRandom``
+            randomly transits between states that have nonzero probability.
+        num_burnin (int): number of steps that will be dropped.
```

Comment:
```suggestion
        num_burnin (int): number of steps that will be dropped. Increasing this value will
        result in a closer approximation but increased runtime.
```
Is this true?

Reply:
This is a good point --- I think "may" is a more accurate wording here.
@multiphaseCFD can you make a follow-up PR?

Reply:
Unless this is already resolved?